### ISSUE TITLE
Add a forgot my password link under the sign-in button and also create a minial forgotmypassword.html page

### ISSUE THREAD

**Issue by monajalal** [davidatbu/openFraming#103]

**Comment by davidatbu** [comment]
Sorry, I closed this because I thought the PR that @edward-edberg  linked addressed this issue. I see @monajalal  that you added this to the Camera Ready milestone.

**Comment by monajalal** [comment]
Yeah, for camera ready, our screenshots for the paper would change to some extent. The reason we shouldn't really work on them is due to waiting on the paper decision on September 15th. I am thinking of having another milestone one for the end of next week and another one for Mid August for the intermediary tasks that need more attention (high and intermediate priorities accordingly). But wasn't sure what would be a good name for them.
